packages/manager/apps/pci-rancher/src/components/layout-helpers/CreateRancher/CreateRancher.component.tsx

```diff
@@ -178,18 +178,7 @@ const CreateRancher: React.FC<CreateRancherProps> = ({
   return (
     <div>
       <Title>{t('createRancherTitle')}</Title>
-
-      <PciDiscoveryBanner project={project} />
-
-      <OsdsMessage
-        color={ODS_THEME_COLOR_INTENT.info}
-        type={ODS_MESSAGE_TYPE.info}
-        className="my-6"
-      >
-        <OsdsText color={ODS_THEME_COLOR_INTENT.text}>
-          <Trans> {t('createRancherInfoMessage')}</Trans> <br />
-        </OsdsText>
-      </OsdsMessage>
+      {isProjectDiscoveryMode && <PciDiscoveryBanner project={project} />}
```

Comment:
```suggestion
      <PciDiscoveryBanner project={project} />
```
adding the conditionnal is not needed anymore, the logic is now part of the banner

Reply:
done